File: app/src/main/java/com/example/unipathapp/AboutProjectActivity.java
package com.example.unipathapp;

import android.widget.ImageButton;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.content.Intent;
public class AboutProjectActivity extends AppCompatActivity {

    ImageButton btnNotification, btnProfile, btnFavorite, btnMain;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about_project);

        btnProfile = findViewById(R.id.btnProfile);
        btnMain = findViewById(R.id.btnMain);
        btnFavorite = findViewById(R.id.btnFavorite);
        btnNotification = findViewById(R.id.btnNotification);

        btnMain.setOnClickListener(v -> {
            startActivity(new Intent(this, MainActivity.class));
        });
        btnFavorite.setOnClickListener(v -> {
            startActivity(new Intent(this, FavoriteActivity.class));
        });
        btnProfile.setOnClickListener(v -> {
            startActivity(new Intent(this, ProfileActivity.class));
        });
        btnNotification.setOnClickListener(v -> {
            startActivity(new Intent(this, NotificationActivity.class));
        });
    }
}

File: app/src/main/java/com/example/unipathapp/AboutProjectActivity.kt
package com.example.unipathapp

import android.content.Intent
import android.os.Bundle
import android.widget.ImageView
import android.widget.LinearLayout
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class AboutProjectActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about_project)
        try {
            val btnBack = findViewById<ImageView>(R.id.btnBackArrow)
            btnBack.setOnClickListener {
                goToProfile()
            }
            val btnMain = findViewById<LinearLayout>(R.id.btnMain)
            btnMain.setOnClickListener {
                startActivity(Intent(this, MainActivity::class.java))
                finish()
            }
            val btnFav = findViewById<LinearLayout>(R.id.btnFavorite)
            btnFav.setOnClickListener {
                startActivity(Intent(this, FavoriteActivity::class.java))
            }
            val btnNotif = findViewById<ImageView>(R.id.btnNotification)
            btnNotif.setOnClickListener {
                startActivity(Intent(this, NotificationActivity::class.java))
            }
            val btnProfileNav = findViewById<LinearLayout>(R.id.btnProfileNav)
            btnProfileNav?.setOnClickListener {
                finish()
            }

        } catch (e: Exception) {
            Toast.makeText(this, "Ошибка: ${e.message}", Toast.LENGTH_LONG).show()
        }
    }

    private fun goToProfile() {
        try {
            val intent = Intent(this, ProfileActivity::class.java)
            intent.flags = Intent.FLAG_ACTIVITY_CLEAR_TOP or Intent.FLAG_ACTIVITY_SINGLE_TOP
            startActivity(intent)
            finish()
        } catch (_: Exception) {
            finish()
        }
    }

}